src/GraphQL/Resolver/Map.php

```diff
@@ -18,7 +18,7 @@ class Map implements ResolverInterface {
   /**
    * Map constructor.
    *
-   * @param $resolver
+   * @param mixed $resolver
```

Comment:
actully this should be `\Drupal\graphql\GraphQL\Resolver\ResolverInterface`

Reply:
please also fix `$resolver` variable above, as it should be also the same type, and not `mixed`